### DISCUSSION TITLE
nestjs-docker-compose-postgres

### DISCUSSION THREAD

**Discussion by giscus** [tomwray13/tomray.dev#70] [Blog Comments]
# nestjs-docker-compose-postgres

Learn how to write a docker-compose file that creates a local environment with hot reloading for NestJS, Postgres and Redis (using Prisma as the ORM layer)

https://www.tomray.dev/nestjs-docker-compose-postgres

**Comment by bam4564** [comment]
Hey there Tom. This was a really helpful blog post, so very appreciative that you took the time to write it. I have a question related to one of the steps you outlined above. 

When creating having the nestjs application container connect to the postgres container, you updated the database connection string so the host was "postgres" instead of "localhost". You mentioned the reason is that this connection string will reference the container for the database, but I'm having trouble finding information online as to why this is the case. Could you explain this in a little more detail?
